webapp/src/client/app/modules/web-client/rdp/web-client-rdp.component.ts

```diff
@@ -337,18 +369,37 @@ export class WebClientRdpComponent extends WebClientBaseComponent implements OnI
   }
 
   private callConnect(connectionParameters: IronRDPConnectionParameters): void {
-    this.remoteClient
-      .connect(
-        connectionParameters.username,
-        connectionParameters.password,
-        connectionParameters.host,
-        connectionParameters.gatewayAddress,
-        connectionParameters.domain,
-        connectionParameters.token,
-        connectionParameters.screenSize,
-        connectionParameters.preConnectionBlob,
-        connectionParameters.kdcProxyUrl,
-      )
+    const configBuilder = this.remoteClient
+      .configBuilder()
+      .withUsername(connectionParameters.username)
+      .withPassword(connectionParameters.password)
+      .withDestination(connectionParameters.host)
+      .withProxyAddress(connectionParameters.gatewayAddress)
+      .withAuthToken(connectionParameters.token);
+
+    if (connectionParameters.domain != null) {
+      configBuilder.withServerDomain(connectionParameters.domain);
+    }
+
+    if (connectionParameters.screenSize != null) {
+      configBuilder.withDesktopSize(connectionParameters.screenSize);
+    }
+
+    configBuilder.withExtension(displayControl(true));
```

Comment:
suggestion: Make this configurable via a checkbox in a follow up PR. (For clarity, the label must be more detailed that "Enable display control").

Reply:
Done in https://github.com/Devolutions/devolutions-gateway/pull/1333